jni/include/commons.h

```diff
@@ -47,10 +47,24 @@ namespace knn_jni {
          * CAUTION: The behavior is undefined if the memory address is deallocated and the method is called
          *
          * @param memoryAddress The address of the memory location where data will be stored.
-         * @param data 2D byte array containing data to be stored in native memory.
+         * @param data 2D byte array containing binary data to be stored in native memory.
          * @param initialCapacity The initial capacity of the memory location.
          * @return memory address of std::vector<uint8_t> where the data is stored.
          */
+        jlong storeBinaryVectorData(knn_jni::JNIUtilInterface *, JNIEnv *, jlong , jobjectArray, jlong, jboolean);
```

Comment:
what is the use use of jboolean parameter in the end? I am not seeing any java docs related to that.

Reply:
updated docs for append